03LessonThree/in-class-project/daily-planet.html

```diff
@@ -3,8 +3,33 @@
     <head>
         <title>The Daily Planet</title>
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
+        <img src="https://laelalon.files.wordpress.com/2013/05/dp-origional-header.jpg" alt="The Daily Planet" width="100%" height:"auto"/>
+        <nav>
+            <a href="#">Sports</a>
+            <a href="#">Local</a>
+            <a href="#">Economy</a>
+        </nav>
     </head>
     <body>
-        <!-- You code start here -->
+      <main>
+        <section>
+        <img src="https://i.kinja-img.com/gawker-media/image/upload/vqdahllvtqadjfjqsfzw.jpg" alt="Superman" width="50%" height="auto"/>
+        <h1>Superman Not So Super</h1>
+        <p>Bacon ipsum dolor amet tenderloin tri-tip hamburger turkey meatloaf. Beef ribs porchetta capicola shoulder pork tail. Turducken shoulder pork jowl pancetta spare ribs, bacon andouille. Cow rump venison kevin pancetta shankle. Pork chop andouille venison tenderloin alcatra filet mignon shank ham landjaeger bacon swine corned beef short ribs brisket porchetta. Turkey porchetta meatloaf picanha filet mignon tail.
+        </p>
+        <p>Chuck doner pig leberkas sausage pastrami pork shoulder shank tongue. Picanha brisket meatloaf pork short ribs shank pork belly tenderloin. Ham hock ham rump strip steak pig pastrami sirloin. Swine short ribs salami, alcatra jowl t-bone bresaola shoulder ham pork sirloin meatloaf spare ribs. Ham ball tip t-bone pig biltong swine beef ribs brisket turkey jowl tri-tip hamburger corned beef. Spare ribs doner ball tip filet mignon brisket cow boudin biltong tongue. Tenderloin boudin corned beef, meatloaf drumstick ham venison.
+        </p>
+        <p>Drumstick tail andouille sirloin ball tip t-bone brisket. Capicola alcatra meatloaf, pig bresaola landjaeger tri-tip fatback bacon andouille meatball jowl. Doner brisket strip steak, ball tip swine pastrami flank filet mignon corned beef chuck bacon ground round shoulder. Leberkas ribeye venison, pancetta cow pork chop andouille ground round corned beef brisket kevin ham flank turkey. Ham hock venison pastrami, salami spare ribs landjaeger sirloin tri-tip filet mignon. Shankle tenderloin ham hock tail filet mignon, biltong brisket kevin cow strip steak.
+        </p>
+      </section>

```

Comment:
You are missing one project requirement: Using the "article" tag. Please update and commit again, and I can bump your grade up to 100%. 

Otherwise, great job! Extra points for the ACA ad!